lib/beaker/hypervisor/vmpooler.rb

```diff
@@ -42,6 +42,10 @@ def load_credentials(dot_fog = '.fog')
     rescue Errno::ENOENT
       @logger.warn "Credentials file (#{dot_fog}) not found; proceeding without authentication"
       creds
+    rescue TypeError
+      #@logger.warn "Errors in credentials file (#{dot_fog}); missing a :default section; proceeding without authentication"
+      @logger.warn "Errors in credentials file "
```

Comment:
@Sukanyade2014 we should choose one of these log lines to keep around, and eliminate the other. I'm thinking that more information is better